EIPS/eip-tbd.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: <to-be-assigned>
+title: Big integer modular exponentiation (EIP-198) gas cost
+author: Kelly Olson (@ineffectualproperty), Sean Gulley (@sean-sn), Simon Peffers (@simonatsn), Justin Drake (@justindrake), Dankrad Feist (@dankrad)
+discussions-to: https://ethereum-magicians.org/t/big-integer-modular-exponentiation-eip-198-gas-cost/4150
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-03-20
+Requires: <EIP-198>
+---
+
+## Simple Summary
+The [EIP-198](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-198.md) ‘big integer modular exponentiation’, or `ModExp`, precompile is currently overpriced. Re-pricing this precompile will enable more cost efficient verification of RSA signatures, verifiable delay functions (VDFs), primality checks, and more.
+
+## Abstract
+After benchmarking the ModExp precompile, we discovered that it is ‘overpriced’ relative to other precompiles. We also discovered that the current gas pricing formula could be improved to better estimate the computational complexity of various ModExp input variables. To improve the gas cost pricing for this precompile the following options are available:
+
+1. Changing the value of the GQUADDIVISOR parameter in the ModExp pricing formula to bring its costs more in-line with other precompiles
+2. Modifying the gas pricing formula to better reflect the computational complexity of ModExp operations
+3. Improving the underlying libraries beneath the ModExp Precompile
+4. Any combination of (1), (2), and (3)
+
+We recommend **Option (1)** which provides a large practical improvement to gas estimation while keeping implementation complexity low. Options (2) and (3) could also be implemented and would further improve the gas pricing for a broader range of use cases. Additional data can be provided for options (2) and (3) as desired.
+
+## Motivation
+Modular exponentiation is a foundational arithmetic operation for many cryptographic functions including signatures, VDFs, SNARKs, accumulators, and more. Unfortunately, the ModExp precompile is currently over-priced, making these operations inefficient and expensive. By reducing the cost of this precompile, these cryptographic functions become more practical, enabling improved security, stronger randomness (VDFs), and more.
+
+## Specification
+The current gas pricing formula is defined in EIP-198. This formula divides a ‘computational complexity’ function by a ‘gas conversion’ parameter called ‘GQUADDIVISOR’ to arrive at a gas cost. 
+
+### **Recommended** Option (1): Change value of GQUADDIVISOR
+`GQUADDIVISOR` is set to ‘20’ per EIP-198. We recommend changing the value of this parameter to ‘200’.
+
+### Option (2): Modify ‘computational complexity’ function 
+A proposed ‘complexity’ function can be found at the following [spreadsheet](https://docs.google.com/spreadsheets/d/1Fq3d3wUjGN0R_FX-VPj7TKhCK33ac--P4QXB9MPQ8iw/edit?usp=sharing).
```

Comment:
An EIP, especially its specification section, should be self contained and not require external links to understand the specification. Is it possible to move in key content from this external link (and keeping it for supporting data)?

Reply:
Thanks for the feedback, added.